kse/src/org/kse/gui/actions/resources_de.properties

```diff
@@ -246,7 +246,7 @@ ExportTrustedCertificatePublicKeyAction.text
 ExportTrustedCertificatePublicKeyAction.tooltip                                  = \u00D6ffentlichen Schl\u00FCssel des vertrauensw\u00FCrdigen Zertifikates exportieren
 
 FindAction.Find.Title       = Finden
-FindAction.NotFound.message = Nicht gefunden
+FindAction.NotFound.message = \u00BB{0}\u00AB Nicht gefunden
```

Comment:
The n of "nicht" should be lower case now

Reply:
You are right.
_But the issue was also before my PR._
I will correct them.